sql/moz-fx-data-shared-prod/firefox_ios_derived/client_adclicks_history_v1/schema.yaml

```diff
@@ -0,0 +1,22 @@
+fields:
+- description: Unique ID for the client installation.
+  mode: NULLABLE
+  name: client_id
+  type: STRING
+- description: Sample ID to limit query results during an analysis.
+  mode: NULLABLE
+  name: sample_id
+  type: INTEGER
+- description: History of ad_clicks for a user, by submission_date
+  fields:
+  - description: The date associated with the ad_click value
+    mode: NULLABLE
+    name: key
+    type: DATE
+  - description: The total number of ad_clicks for the submission_date
+    mode: NULLABLE
+    name: value
+    type: INTEGER
```

Comment:
nit: I assume the udf specifically creates a struct with `key` and `value` as attributes. Would it be worth renaming these to something more descriptive? Like `date` and `ad_clicks`?